src/ensembl/src/content/app/custom-download/components/checkbox-with-selects/CheckboxWithSelects.tsx

```diff
@@ -0,0 +1,207 @@
+import React, { useState, useEffect, useCallback } from 'react';
+import Checkbox from 'src/shared/checkbox/Checkbox';
+import Select, { Option } from 'src/shared/select/Select';
+import styles from './CheckboxWithSelects.scss';
+
+import ImageButton from 'src/shared/image-button/ImageButton';
+import { ReactComponent as AddIcon } from 'static/img/browser/zoom-in.svg';
+import { ReactComponent as RemoveIcon } from 'static/img/shared/clear.svg';
+
+type Props = {
+  selectOptions: Option[];
```

Comment:
perhaps just call this prop `options` instead of `selectOptions`? A bit hard to tell the difference between `selectOptions` and `selectedOptions`.

Reply:
That's done